lib/src/main/kotlin/io/hammerhead/karooext/models/KarooEffect.kt

```diff
@@ -236,3 +264,27 @@ data class ApplyLauncherBackground(
      */
     val url: String?,
 ) : KarooEffect()
+
+/**
+ * Switch the currently visible in-ride page to the map, optionally changing zoom if already on map
+ */
+@Serializable
+data class ShowMapPage(
+    /**
+     * If already on the map page, should the map zoom level be toggled
```

Comment:
Could you clarify the `map zoom level be toggled`? 

Does this reset the map zoom to the default? Or does it toggle the display of the other ride elements over the map? (that could be a cool feature to have 👀)

Reply:
The map page zooms through zoom levels (13, 15, 16). This is accessed normally through the bottom-left key button (magnifying glass with +) on the map page or by ANT+ function set control for "Cycle Map Zoom".